docs/guide/sections/dom-binding.md

```diff
@@ -1,6 +1,71 @@
-Full documentation for the DOM Binding module is coming soon.
+Cart.js ships with support for [Rivets.js][], which allows for some powerful templating approaches with minimal effort. Rivets.js allows you to describe in your markup how your data should be rendered, and then handles automatically updating the DOM when that model data changes. With Cart.js, this allows us to describe a template for our cart that will be automatically updated whenever the cart changes.
 
-In the meantime, you can check out the example on the [front page][] in combination with the [Rivets.js documentation][] to get an idea of how things work.
+You can see an example of this in action [at the end of this section].
+
+### Getting Started
+The Rivets module of Cart.js will bind to an element with the attribute `data-cart-view="data-cart-view"`, which needs to wrap all cart elements that need to be dynamically updated. The basic syntax for Rivets templates involves adding an `rv-` attribute to the element, and passing in a reference to the `cart` object that Cart.js has bound with Rivets. 
```

Comment:
Could we rephrase this to be more active and focus on what the user needs to do, rather than descriptive of what Cart.js does? So something more along the lines of "to get started with the DOM Binding module, first add a `data-cart-view` attribute to a parent element of any content you'd like to be automatically updated when the cart changes"... "you can then use one a number of `rv-*` attributes to describe how HTML should be rendered from your cart data".

A couple of practical `rv-` examples would be good, too.